### ISSUE TITLE
Better Task Creation Page

### ISSUE THREAD

**Issue by Lauritz-Tieste** [Bootstrap-Academy/Bootstrap-Academy#151]
![Image](https://github.com/user-attachments/assets/e44f3e8e-0f45-489c-aed1-293b5a15d694)

1. better create button with better title and icon
2. overview of how many quizzes exist and how many have been created by you
3. marking when the quiz was created by you
4. if not created by you, name the author
5. describe the type of quiz (single-choice/multi-choice)
6. delete and edit icon only if user is admin; otherwise eye icon

Page: http://test.bootstrap.academy/quizzes/start/medienkompetenz/create ...

**Comment by Mert-55** [comment]
Hi, I would like to take over this issue.

**Comment by Lauritz-Tieste** [comment]
It was discussed internally that this ticket would be put on hold for now, as there have been problems recently with quizzes that were not of the required quality. In addition, this ticket does not have a bounty yet. (@TheMorpheus407)

**Comment by TheMorpheus407** [comment]
That's right. We need to fix that before we move on with user generated quizzes

**Comment by rumpel360** [comment]
Issues with low quality quizzes might be enhanced if there were some quality criteria listed during the creation process.
By the way, few days ago I realized that the Buttons content changes from "als erledigt markieren" to "Aufgabe hinzufügen" without further notice after I marked a topic as done. As I didn't understand why I just tried out and more or less accidently started a review process (I guess) on a medaningless question. No idea how oftent that happens, but hint like "please don`t send trash" before submitting might avoid that you guys have to waste your time.
I also suggest to change the Button Text from "Aufgabe hinzufügen" to "Neue Quizz-Frage einreichen" to make its effect more clear.

**Comment by NilocLP** [comment]
> It was discussed internally that this ticket would be put on hold for now, as there have been problems recently with quizzes that were not of the required quality. In addition, this ticket does not have a bounty yet. ([@TheMorpheus407](https://github.com/TheMorpheus407))

As a solution, you could make it so that questions are not directly visible, but are first unpublished and other users who have a certain level in the skill of the question can then check the question and then unlock or remove it. The creator may not review his own question and must have completed the lesson for which he wants to create a question.

Morphcoins could also be awarded as an incentive. For example, the creator of the question receives 3 Morphcoins for creating the question once it has been released. The reviewer receives 5 Morphcoins when he reviews a question, regardless of whether he releases or removes it. 

Example:
- Level required to review questions: 10

1. x creates a question on the topic of HTML
2. Y (level 10 in web development) can see that there are unpublished questions.
3. Y reviews X's task and realizes that the task is correct. He can now publish it. He receives 5 coins for his effort
4. x now receives 3 coins and his question is available to everyone.


Bonus: To avoid trolls, you could appoint moderators who can delete published questions and exclude users who review incorrectly too often from the review feature. 
However, it should be noted that some skill sections have too few lessons to reach a certain level and people can currently simply skip through all courses without viewing them in order to “farm” levels

@Lauritz-Tieste

**Comment by Lauritz-Tieste** [comment]
@TheMorpheus407

**Comment by TheMorpheus407** [comment]
Yes that might actually be the best solution. 
I think the following might be suitable:

- Allow everyone to create quizzes
- have moderators (appointed) or people with level >= 10 in this specific skill accept or dismiss new questions
- Option to report quizzes (like we have at the moment) stays. Reported questions can also be edited and reviewed by mods
- Accepting, Moderating, Modifying, ... will earn a moderator or people with level >= 10 in this specific skill 5 coins
- Writing a quiz questions will award the creator of such question also 5 coins at the moment it is accepted by a moderator or people with level >= 10 in this specific skill

**Comment by TheMorpheus407** [comment]
Therefore we need 

- new views of the admin dashboard for moderators (for point 3)
- new roles
- new state for tasks in general to be reviewed
- the actual review possibility
thats quite a lot
